README.md

```diff
@@ -0,0 +1,55 @@
+# Explorations for A Sustainable Future
+
+**A place for exploratory and micro data science projects for Nesta's Sustainable Future Mission**
+
+This repository is for work that is very small and self-contained, is not part of a specific Sustainable Future project or is exploratory in nature. Examples of what might be included are exploring the potential for a new dataset, testing out a modelling approach or handling an ad-hoc request for data analysis.
+
+Each exploration is in its own sub-directory, which can be navigated using the Directory below. Contributing guidelines are below.
+
+## 📖 Directory
+
+**< Project name (with link to sub-directory) >**
+< Short description of project and links to any key resources >
+
+
+## 🎮 User guidelines
+
+1. Use the directory above to find the exploration that you are looking for.
+2. Create an environment for that exploration. This will probably involve running `conda create -n <exploration>_<name> python=3` and installing some requirements, but check each exploration for its installation instructions.
+
+## 📝  Contributor guidelines
+
+To add a new exploration to this repository:
+
+1. Determine whether this is the right place for the work. As a rule of thumb it should be:
+  a. Executed with a fairly small codebase (e.g. 1 notebook and a utils module).
```

Comment:
So each exploration folder with the repo should not follow the cookiecutter structure? Where should outputs be stored (such as figures)? Should the utils .py file have data getters together will other necessary functions?

Reply:
> So each exploration folder with the repo should not follow the cookiecutter structure? 

This won't be a requirement, but there's no reason why someone couldn't deploy the cookiecutter structure in subdir. I don't really want to encourage it though as I think that once you're in the territory of having multiple modules, and various pipeline producing many outputs, you have probably outgrown this repo.

> Where should outputs be stored (such as figures)? 

I was going to leave this up to the developer, but on reflection, we could provide a very lightweigh template subdir containing a readme template, plus a minimal folder structure. What do you think?

> Should the utils .py file have data getters together will other necessary functions?

Yep. I don't see a problem with e.g. separating basic utils and exploration-specific functions into two separate modules. What I think would be better is if we start to add commonly used functions to a separate `ds-utils` repo that can be pip installed. There are several candidates for this already, such as reading files from S3, downloading files from the web, basic text processing etc.

Reply:
I like the idea of the template subdirectory, would love to get started on `ds_utils` too

Reply:
Sounds good!